requirements-dev.txt

```diff
@@ -29,9 +29,9 @@ cloudpickle
 rpyc==4.1.5
 scikit-learn
 git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e0a3c380e9
-# TODO(https://github.com/modin-project/modin/issues/5171): uncap xgboost
-# when we can run tests on 1.7.0 without getting stuck.
-xgboost>=1.6.0,<1.7.0
+# TODO(https://github.com/modin-project/modin/issues/5194): Uncap xgboost
+# when we use collective instead of rabit.
+xgboost>=1.6.0,<2.0.0
```

Comment:
This feels wrong, it should probably be `>=1.7.1`, as 1.7.0 is broken for us and a fix is made in 1.7.1...

Reply:
Now using xgboost>=1.7.1,<2.0.0 everywhere